distributionResources/embedded/config/application.properties

```diff
@@ -0,0 +1,163 @@
+server.port=8080
+
+## To use ssl, update the properties below for your local installation
+
+#server.ssl.enabled=true
+#server.ssl.enabled-protocols=TLSv1.3,TLSv1.2,TLSv1.1
+#server.ssl.protocol=TLS
+#server.ssl.key-alias=tomcat
+#server.ssl.key-store=@@keyStore@@
+#server.ssl.key-store-password=@@keyStorePassword@@
+## Typically either PKCS12 or JKS
+#server.ssl.key-store-type=PKCS12
+#server.ssl.ciphers=HIGH:!ADH:!EXP:!SSLv2:!SSLv3:!MEDIUM:!LOW:!NULL:!aNULL
+
+## HTTP-only port for servers that need to handle both HTTPS (configure via server.port and server.ssl above) and HTTP
+#context.httpPort=8080
+
+## Database connections. All deployments need a labkeyDataSource as their primary database. Add additional external
+## data sources by specifying the required properties (at least driverClassName, url, username, and password)
+## with a prefix of context.resources.jdbc.<dataSourceName>.
+context.resources.jdbc.labkeyDataSource.type=javax.sql.DataSource
+context.resources.jdbc.labkeyDataSource.driverClassName=org.postgresql.Driver
+context.resources.jdbc.labkeyDataSource.url=jdbc:postgresql://localhost/labkey
+context.resources.jdbc.labkeyDataSource.username=@@jdbcUser@@
+context.resources.jdbc.labkeyDataSource.password=@@jdbcPassword@@
+context.resources.jdbc.labkeyDataSource.maxTotal=50
+context.resources.jdbc.labkeyDataSource.maxIdle=10
+context.resources.jdbc.labkeyDataSource.maxWaitMillis=120000
+context.resources.jdbc.labkeyDataSource.accessToUnderlyingConnectionAllowed=true
+context.resources.jdbc.labkeyDataSource.validationQuery=SELECT 1
+#context.resources.jdbc.labkeyDataSource.logQueries=true
+#context.resources.jdbc.labkeyDataSource.displayName=Alternate Display Name
+
+#context.resources.jdbc.@@extraJdbcDataSource@@.driverClassName=@@extraJdbcDriverClassName@@
+#context.resources.jdbc.@@extraJdbcDataSource@@.url=@@extraJdbcUrl@@
+#context.resources.jdbc.@@extraJdbcDataSource@@.username=@@extraJdbcUsername@@
+#context.resources.jdbc.@@extraJdbcDataSource@@.password=@@extraJdbcPassword@@
+
+context.encryptionKey=@@encryptionKey@@
+
+## By default, we deploy to the root context path. However, some servers have historically used /labkey or even /cpas
```

Comment:
Since this is to be included with a distribution, I think this comment should be updated to instruct users on intended usage, not historical context.